versioned_hdf5/tests/test_api.py

```diff
@@ -1168,6 +1224,43 @@ def test_multidimensional():
         assert version2['test_data'][0, 1] == 2
         assert_equal(version2['test_data'][()], data2)
 
+        data3 = data.copy()
+        data3[0:1] = 3
+
+        with file.stage_version('version3', 'version1') as g:
```

Comment:
Is there a special reason why you used `version1` as the previous version? Just out of curiosity

Reply:
If you don't specify a previous version, it defaults to the most recently created version, which in this case would be version2. However, want to test against modifications from the base data, which is in version1. Otherwise I would have to also include the modifications that I did in version2 above, which would make the test more complicated (admittedly not too much more complicated, but it's easier to see what's going wrong when things break if you don't have to recreate several layers of changes).